presto-hive/src/test/java/io/prestosql/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -2285,6 +2286,29 @@ public void testCreateExternalTableTextFileFieldSeparatorEscape()
         deleteRecursively(tempDir.toPath(), ALLOW_INSECURE);
     }
 
+    @Test
+    public void testCreateExternalTableWithCTASWritesDisabled()
+            throws IOException
+    {
+        File tempDir = createTempDir();
+        File dataFile = new File(tempDir, "test.txt");
+
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s.%s.test_create_external " +
+                        "WITH (\n" +
+                        "   external_location = '%s',\n" +
+                        "   format = 'TEXTFILE',\n" +
+                        "   textfile_field_separator = U&'\\0001'\n" +
+                        ") AS " +
+                        "SELECT * FROM orders LIMIT 5",
+                getSession().getCatalog().get(),
+                getSession().getSchema().get(),
+                new Path(tempDir.toURI().toASCIIString()).toString());
+
+        assertQueryFails(createTableSql, "External tables cannot be created using CREATE TABLE AS");
```

Comment:
I think we should also update the message, to indicate that this is not a permanent condition and may be subject to configuration change

ie change
```
External tables cannot be created using CREATE TABLE AS
```

to

```
Cannot create a non-managed Hive table using CREATE TABLE AS
```